packages/api/test/util/describeE2E.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+const WS_ENDPOINTS = {
+  'local': 'ws://127.0.0.1:9944/',
+  'substrate-master': 'ws://127.0.0.1:9945/',
+  'substrate-1.0': 'ws://127.0.0.1:9946/',
+  'substrate-2.0': 'ws://127.0.0.1:9947/',
+  'polkadot-master': 'ws://127.0.0.1:9948/',
+  'polkadot-alexander': 'ws://127.0.0.1:9949/',
+  'remote-polkadot-alexander': 'wss://poc3-rpc.polkadot.io/',
+  'remote-substrate-1.0': 'wss://substrate-rpc.parity.io/'
+};
+
+type WsName = keyof typeof WS_ENDPOINTS;
+
+interface Options {
+  except?: WsName[]; // Only one of the two keys `either` `only` should be set
+  only?: WsName[];
+  apiType?: 'promise' | 'rxjs';
+}
+
+// From the options and the WITH_DOCKER flag, calculate on which endpoints we
```

Comment:
Possible to have a `"test:docker": "WITH_DOCKER yarn test packages/api/test/e2e"` command?

- no coverage
- only runs the e2e stuff (assuming all in the same location)